packages/app-desktop/gui/NoteRevisionViewer.tsx

```diff
@@ -92,6 +93,7 @@ const NoteRevisionViewerComponent: React.FC<Props> = ({ themeId, noteId, onBack,
 
 		setRevisions(revisions);
 		setCurrentRevId(revisions.length ? revisions[revisions.length - 1].id : '');
+		focus('NoteRevisionViewer::viewer_domReady', viewerRef.current);
```

Comment:
Thank you for working on this! To prevent the autofocus from being removed in the future, consider adding a comment explaining why this autofocus is necessary.

**(Optional)**: It may make more sense for focus to initially jump to the revisions dropdown menu.